src/core/database/transactional-mutations.interceptor.ts

```diff
@@ -28,9 +28,8 @@ export class TransactionalMutationsInterceptor implements NestInterceptor {
       return next.handle();
     }
 
-    const res = await this.db.runInTransaction(
```

Comment:
What's the effective difference between the `of` vs `from`?

Reply:
The only one I can think of is this method will throw an exception instead of having the returned observable emit the error. The caller of this lifecycle hands this, so it wasn't a problem. This just felt cleaner.
```diff
- observable -> promise -> awaited -> observable / thrown error
+ observable -> promise -> observable
```

Reply:
👌🏼